Mathlib/RingTheory/Ideal/Quotient.lean

```diff
@@ -215,17 +215,29 @@ theorem exists_inv {I : Ideal R} [hI : I.IsMaximal] :
 
 open Classical
 
-/-- quotient by maximal ideal is a field. def rather than instance, since users will have
-computable inverses in some applications.
+/-- The quotient by a maximal ideal is a group with zero. This is a `def` rather than `instance`,
+since users will have computable inverses in some applications.
+
 See note [reducible non-instances]. -/
 @[reducible]
-protected noncomputable def field (I : Ideal R) [hI : I.IsMaximal] : Field (R ⧸ I) :=
+protected noncomputable def groupWithZero (I : Ideal R) [hI : I.IsMaximal] :
+    GroupWithZero (R ⧸ I) :=
   { Quotient.commRing I,
-    Quotient.isDomain I with
+    Quotient.isDomain
+      I with
     inv := fun a => if ha : a = 0 then 0 else Classical.choose (exists_inv ha)
-    mul_inv_cancel := fun a (ha : a ≠ 0) =>
-      show a * dite _ _ _ = _ by rw [dif_neg ha]; exact Classical.choose_spec (exists_inv ha)
+    mul_inv_cancel := fun a (ha : a ≠ 0) ↦
+      show a * dite _ _ _ = _ by rw [dif_neg ha] ; exact Classical.choose_spec (exists_inv ha)
```

Comment:
These changes don't seem necessary to me; can you revert them?

Reply:
Sure, but Itthought using the ascii thing was the correct mathlib4 style

Reply:
I think we decided not to change it during (forward) porting as it just adds noise. I don't think spaces around `;` is preferred at all.